### DISCUSSION TITLE
写下2.0,你认为需要优先支持的组件

### DISCUSSION THREAD

**Discussion by xiaoyatong** [jdf2e/nutui-react#942] [Q&A]
感谢大家，辛苦反馈。

目前2.0 alpha 版本已发布组件：

Button | 按钮组件
-- | --
Cell | 单元格组件
Image | 图片
OverLay | 遮罩层
Divider | 分割线
Grid | 宫格
Layout | 布局
Sticky | 粘性布局
BackTop | 返回顶部
Elevator | 电梯楼层
FixedNav | 悬浮导航
Navbar | 导航组件
InputNumber | 数字输入框
Menu | 菜单
Rate | 评分
Uploader | 上传组件
Badge | 徽标
Drag | 拖拽
Popup | 弹出层
Animate | 动画/动效
Audio | 音频播放器
Indicator | 指示器
Pagination | 分页
Progress | 进度条

**Comment by xiaoyatong** [comment]
预计完成：


Tabbar | 标签栏组件
-- | --
Tabs | 选项卡切换
Input | 输入框
Switch | 开关
Dialog | 对话框

**Reply by zbw920529** [reply]
类似于antd 的Form表单，支持分组，支持数组，比如：name设置为['user','hobby']，最后数据结构为:{user: {hobby: '看电影',}}

**Reply by xiaoyatong** [reply]
好的

**Comment by YuDaGuo888** [comment]
Collapse 折叠面板 titleIcon 自定义图片

**Reply by xiaoyatong** [reply]
跟进中。

**Comment by Xiakeming97** [comment]
![单个菜单](https://user-images.githubusercontent.com/111333326/233979148-23ff206a-2928-402d-b962-05ccd46ff2ac.png)
在单个菜单时希望增加一个可以选择位置的变量
如 位置 position：left ｜content ｜right
- 需求是这样的
![需求图片](https://user-images.githubusercontent.com/111333326/233980648-2d527a4f-bd4d-4c1f-aa9e-3582d7db1c45.png)

**Reply by xiaoyatong** [reply]
放入menu的迭代里跟进。

**Comment by Me-UserName** [comment]
Space、Card、Dropdown、Calendar、Steps、Form

**Reply by xiaoyatong** [reply]
steps 正在处理。接下来会是popover（dropdown）、 calendar、form。其他我理解目前还没有，需新增。

**Comment by vdfor** [comment]
Swipe 组件，并且希望在Taro下能够支持在自定义组件（CustomWrapper）下使用，以提升性能。

**Reply by xiaoyatong** [reply]
好的，我们考虑一下。

**Comment by suaohub** [comment]
希望能够新增 Form组件，并且能够支持类似antd proComponents 中的 ProFormDependency 数据联动功能，毕竟表单数据联动场景还是比较多 ；swipe 体验不是很好，有的时候划不动 官方demo 就看的出来

**Reply by xiaoyatong** [reply]
好的。

**Comment by jiangshengdev** [comment]
https://element.eleme.cn/#/zh-CN/component/form

https://ant.design/components/form-cn

表单组件，在中后台小程序中被大量使用，期望能够有（部分）像上面2个链接中的表单组件类似的表单验证能力

期望能支持在一个form item 中验证多个表单元素（如 input radio checkbox等）

期望能在修改表单元素时，及时更新验证状态，及时显示新的错误文本（如input 的blur触发之后，radio的change触发之后），而不是点击提交按钮之后

期望能够支持表单嵌套

**Comment by lhc1995** [comment]
需要优化picker组件 下拉选项很多 滚动卡顿 以及需要做个搜索功能
